src/Http/Livewire/Models/CategoryForm.php

```diff
@@ -64,6 +67,11 @@ public function getModel(): Model
         return $this->category;
     }
 
+    public function articles()
+    {
+        return Article::where('category_id', $this->category->id)->paginate(8);
```

Comment:
To keep the abstraction as high as possible I would suggest to use the relationship already defined on the Article model. `$this->category->articles()->paginate(8);`